api/src/main/java/org/pentaho/platform/api/genericfile/model/IGenericFileContentWrapper.java

```diff
@@ -0,0 +1,13 @@
+package org.pentaho.platform.api.genericfile.model;
+
+import java.io.InputStream;
+
+public interface IGenericFileContentWrapper {
+  InputStream getInputStream();
+  String getFileName();
+  String getMimeType();
+
+  void setInputStream( InputStream inputStream);
+  void setFileName(String fileName);
+  void setMimeType(String mimeType);
```

Comment:
Are the setters really needed on the interface?

Reply:
no, I will remove them...